build/dependencies.props

```diff
@@ -31,6 +31,7 @@
     <SystemThreadingTasksDataflowPackageVersion>4.9.0-preview3-26423-04</SystemThreadingTasksDataflowPackageVersion>
     <SystemThreadingTasksExtensionsPackageVersion>4.5.0-preview3-26423-04</SystemThreadingTasksExtensionsPackageVersion>
     <SystemValueTuplePackageVersion>4.5.0-preview3-26423-04</SystemValueTuplePackageVersion>
+    <SdkPackageVersion>2.1.300-preview2-008530</SdkPackageVersion>
```

Comment:
I don't believe we can put this here since ProdCon doesn't flow an SDK version to us. We build before the CLI, so this wouldn't be available anyways.

Reply:
Fair duce.